ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryControllerTest.java

```diff
@@ -228,28 +228,40 @@ public void bootstrapPeersPongReceived_HashMatched() {
         .hasSize(3);
 
     // Simulate a PONG message from peer 0.
-    final PongPacketData packetData =
-        PongPacketData.create(localPeer.getEndpoint(), mockPacket.getHash());
-    final Packet packet = Packet.create(PacketType.PONG, packetData, keyPairs.get(0));
-    controller.onMessage(packet, peers.get(0));
+
+    for (int i = 0; i < 3; i++) {
+      final PongPacketData packetData =
+          PongPacketData.create(localPeer.getEndpoint(), mockPacket.getHash());
+      final Packet packet0 = Packet.create(PacketType.PONG, packetData, keyPairs.get(i));
+      controller.onMessage(packet0, peers.get(i));
+    }
 
     // Ensure that the peer controller is now sending FIND_NEIGHBORS messages for this peer.
     verify(outboundMessageHandler, times(1))
```

Comment:
You can loop over your peers here (and below) instead of explicitly checking each one by hard-coded index